webapp/migrations/Version20240511091916.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240511091916 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fill default version command for compiler/runner.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('UPDATE language SET compiler_version_command = \'gnatmake --version\' where langid=\'adb\' AND compiler_version_command IS NULL;');
```

Comment:
For easier readability, I would suggest creating an array something like this:

```
$versionCommands = [
'adb' => 'gnatmake --version',
'awk' => 'awk --version',
...
];
foreach ($versionCommands as $lang => $versionCommand) {
   $this->addSql("UPDATE language SET compiler_version_command = '" . $v ."' WHERE langid='" . $lang . "' AND compiler_version_command IS NULL;')
}
```

And then the same thing for runner versions.

Reply:
Done